src/NosCore.Core/Networking/WebApiAccess.cs

```diff
@@ -80,5 +84,94 @@ public T Get<T>(string route, ServerConfiguration webApi = null)
 
             throw new HttpRequestException(response.Headers.ToString());
         }
+
+        public T Post<T>(string route, object data, ServerConfiguration webApi = null)
+        {
+            if (MockValues.ContainsKey(route))
+            {
+                return (T)MockValues[route];
+            }
+
+            var client = new HttpClient();
+            HttpResponseMessage response;
+            client.BaseAddress = webApi == null ? BaseAddress : new Uri(webApi.ToString());
+            if (Token == null)
+            {
+                response = client.PostAsync("api/token/connectserver", Content).Result;
+                if (response.IsSuccessStatusCode)
+                {
+                    Token = response.Content.ReadAsStringAsync().Result;
+                }
+                else
+                {
+                    throw new HttpRequestException(response.Headers.ToString());
+                }
+            }
+
+            var param = new Dictionary<string, string>
```

Comment:
https://stackoverflow.com/questions/1207731/how-can-i-deserialize-json-to-a-simple-dictionarystring-string-in-asp-net

Reply:
Not possible because I need the keys of this dictionary, so I need : 

"postedData" & the Json string